### PR TITLE
Implements `@radial-gradient(circle, ...)`

### PR DESCRIPTION
**Pull request by ogoffart**
Only the circle is implemented so far.

Part of #263

### CODE REVIEW

**Review by tronical — APPROVED** [review]
Great!

(I guess the fix for `semantic_tokens.rs` is a simple replace?)

**Comment by ogoffart** [comment]
> (I guess the fix for semantic_tokens.rs is a simple replace?)

Yes

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ All notable changes to this project are documented in this file.
 ### Added
 
  - Added the Model Adapters `FilterModel` and `MapModel`.
+ - Added `@linear-gradient(circle, ...)`
```
Comment (levrik) [inline CHANGELOG.md:13]: typo `linear` -> `radial`

Comment (tronical) [reply CHANGELOG.md:13]: Well spotted :). Fixed.

**Pull request merged**